### DISCUSSION TITLE
Images continued

### DISCUSSION THREAD

**Discussion by cyberguerilla68** [welpo/tabi#419] [Q&A]
Hi, I was able to add an image to the post using the formatting that you gave me in the previous thread. Is it possible to display an image under the title and between the dates on posts without changing the html? I would like it to be like how proejcts are formatted, such that they show up on the list of blogposts, and in 'recent posts.'

**Comment by welpo** [comment]
It's not possible, no.

It was requested first in #181, but no PR ever happened.

I would take a PR for this change, though. And, of course, I'd be happy to offer feedback and guidance as needed.

**Reply by cyberguerilla68** [reply]
I added it to the html, I put this between date and title in page.html
```
   {#- Image -#}
{% if page.extra.header_image %}
    <img src="{{ page.extra.header_image }}" alt="Header image for {{ page.title }}">
{% endif %}
```
Only issue is that after adding a couple paragraphs (or another image), it breaks scrolling and cuts off the header.

I might try to get it so that it shows in the post listing too. Would I edit the list_posts.html template?

**Reply by welpo** [reply]
What does it look like? Could you show a screenshot?

**Reply by cyberguerilla68** [reply]
Sure thing!

With text:

![image](https://github.com/user-attachments/assets/50e79fa4-8fad-4b8f-b3bd-f3623884c19f)

Without text:

![image](https://github.com/user-attachments/assets/4d530d60-71be-410e-9b3d-ccc89789afa1)

**Reply by welpo** [reply]
Ah, I see! I thought you meant in the main page or in the `blog/` directory.
